### DISCUSSION TITLE
Argument syntax

### DISCUSSION THREAD

**Discussion by lonix1** [containrrr/watchtower#1714] [Q&A] [answered]
The [arguments page](https://containrrr.dev/watchtower/arguments) shows a variety of args. I'm unsure of the expected syntax.

For some args, obviously the value must be specified, e.g.:
```yml
command:
  - --log-level debug
```

But for others, should I specify this:
```yml
command:
  - --cleanup
  - --label-enable
```

Or this:
```yml
command:
  - --cleanup true
  - --label-enable true
```

Is the value expected, or inferred by the existence of the switch itself?

**Comment by piksel** [comment, marked as answer]
Boolean flags should never have an argument when passed as arguments/command, it's only for when the options are set using environment variables.

So, never pass `true` as a command line argument, the presence of the flag is what enables the option.

**Reply by lonix1** [reply]
Thanks for confirming!
